### PR TITLE
Solving issue #98

### PR DESCRIPTION
**Pull request by ghost**
Hi Sami,

- Added url link to each one of the logos in logos.json. 
   Used a small web scrapper in order to populate the url property so there might be minor 
   mistakes or cases when the library may have a website but the github repo came first in google 
   search. 
   (What I did is to google search each logo with node scrapper. The search query was 
   "logo.name + js + javascript" and I grabbed the first result so most of the links should be right)

- Added a section in which the user get's the right answer for the wrong guessed logo + a 
  suggestion to learn more about he library.

- Now JsLogo size can be controlled with a size prop. (I had to do it so I can give it a different size 
  on the result page.

### CODE REVIEW

**Comment by ghost** [comment]
I think the conflict is because I did not have Dutier when I worked on this PR.
The link for Dutier is : https://ecomfe.github.io/echarts/index-en.html .

**Comment by ghost** [comment]
Finished now going over the logos.
Corrected some of the links.
Also there are 8 logos that I tried to find manually and did not find any results. 
If they can not be found on google I think maybe they should not appear here.
The list is:
- Generact
- Generate
- JSTransformer
- React-Sizes
- Toolkit
- Update
- Wiretie
- Titanium (Looks like it part of http://www.appcelerator.com/ but is not present as individual js library/framework)

**Comment by ghost** [comment]
I shouldn't have rushed to remove logos so fast.
Went again over the list of logos stated above.
Managed to find pages for React-Sizes, Generact, JSTransformer and Wiretie.
Still cant find pages (that include the logos) of Generate, Toolkit, Update.
I suggest that if the "learning suggestion" part will be added, it should link to a page that contains the logo associated with the library.

**Comment by samiheikki** [comment]
Hi @maxvcode ! Really **AWESOME** PR 🎉 🎉 🎉 , will certainly merge this PR once it can be merged.

I found URLs for the logos that you were missing.
- Generate https://github.com/generate
- Toolkit https://github.com/node-toolkit
- Update http://update.github.io/update/ 

Just say if you have any other URLs missing and will help you finding them. I'll later today go through the PR and give comments if something needed to be changed. But quickly looking everything looks great. I tried the app and the UI looks perfect.

**Comment by samiheikki** [comment]
I will also halt merging new logos for a while, so we can get this feature out without extra rebasing.

File: src/components/ResultPage.vue
Comment (samiheikki) [inline src/components/ResultPage.vue]: typo here `.learning-suggetsion`

**Comment by samiheikki** [comment]
Let's make sure that every logo will stay in there, so again, just say if you need help finding the URLs for logos.

**Comment by ghost** [comment]
Hi Sami,

- Corrected the typo.
- Added the logos that I removed in the previous commit + added Dutier that was missing.
- Changed "feedback" computed property to "insult" (after all we are using the insults array and we display an insult)
- Added computed property and styling to apply font size change to long insults (if not applied for long insults the design breaks and the text goes to the top of the screen)
 
I'm not sure why it says there are conflicts because I have added Dutier to logos.json (I think that the previous conflict was about it missing). 

It says : "These conflicts are too complex to resolve in the web editor"

File: src/components/JsLogo.vue
Comment (samiheikki) [inline src/components/JsLogo.vue]: Can you change `Learn about this logo library` to `Learn about this JavaScript library`

File: src/components/ResultPage.vue
Comment (samiheikki) [inline src/components/ResultPage.vue]: Is there some use for `ref="insult"` here?

File: src/components/ResultPage.vue
Comment (samiheikki) [inline src/components/ResultPage.vue]: Can you set `<strong>{{currentLogo.name}}!</strong>` to be `<a v-bind:href="currentLogo.url" target="_blank"><strong>{{currentLogo.name}}!</strong></a>`. On my initial attempt, I was trying to click it and thought that to be the URL. I think both the logo and the text can be links.

Also checked out that it will require a bit of CSS changes. Can you also make sure that it has 
```
color: #66BB6A;
text-decoration: none;
```

**Review by samiheikki — CHANGES_REQUESTED** [review]
Great code! Just have a couple of comments and after resolving those it is ready to be merged.

Conflicts seem to be too complicated for GitHub tool so a manual rebase is needed. You can do that either with git or for example using SourceTree.

If you have troubles making the rebase, I can do it also. But in that case I'm unable to modify your original PR and would have to make a new one. But luckily your commits and contribution history will stay there.

Let me know how you want to proceed! Cheers 🍻

**Comment by ghost** [comment]
Made the rebase, now there aren't any conflicts.
Applied the changes requested (the ref="insult" is something I forgot to remove in previous commit).
Thank you for the code review :smiley:

File: src/components/ResultPage.vue
Comment (samiheikki) [inline src/components/ResultPage.vue]: Add `v-if="answerCount !== amount"` so that `learning-suggestion` is not shown if all the logos were answered correctly.

**Review by samiheikki — CHANGES_REQUESTED** [review]
Just one more change needed before merging 🎉

File: src/components/ResultPage.vue
Comment (ghost) [reply src/components/ResultPage.vue]: @samiheikki 
Thanks for pointing it out.
Fixed it.

**Review by samiheikki — APPROVED** [review]

**Comment by samiheikki** [comment]
Perfect! Thank you! :tada:

**Comment by ghost** [comment]
@samiheikki  
Thanks for accepting the changes and for helping me out with code reviews :smiley:

**Pull request merged**
